devito/symbolics/extended_sympy.py

```diff
@@ -374,77 +372,3 @@ class Literal(sympy.Symbol):
 FLOOR = Function('floor')
 
 cast_mapper = {np.float32: FLOAT, float: DOUBLE, np.float64: DOUBLE}
-
-
-class taylor_sin(TrigonometricFunction):
-
-    """
-    Approximation of the sine function using a Taylor polynomial.
-    """
-
-    @classmethod
-    def eval(cls, arg):
-        return eval_taylor_sin(arg)
-
-
-class taylor_cos(TrigonometricFunction):
-
-    """
-    Approximation of the cosine function using a Taylor polynomial.
-    """
-
-    @classmethod
-    def eval(cls, arg):
-        return 1.0 if arg == 0.0 else eval_taylor_cos(arg)
-
-
-class bhaskara_sin(TrigonometricFunction):
-
-    """
-    Approximation of the sine function using a Bhaskara polynomial.
-    """
-
-    @classmethod
-    def eval(cls, arg):
-        return eval_bhaskara_sin(arg)
-
-
-class bhaskara_cos(TrigonometricFunction):
-
-    """
-    Approximation of the cosine function using a Bhaskara polynomial.
-    """
-
-    @classmethod
-    def eval(cls, arg):
-        return 1.0 if arg == 0.0 else eval_bhaskara_sin(arg + 1.5708)
-
-
-# Utils
-
-def eval_bhaskara_sin(expr):
-    return 16.0*expr*(3.1416-abs(expr))/(49.3483-4.0*abs(expr)*(3.1416-abs(expr)))
-
-
-def eval_taylor_sin(expr):
-    v = expr + Mul(-1/6.0,
-                   Mul(expr, expr, expr, evaluate=False),
-                   1.0 + Mul(Mul(expr, expr, evaluate=False), -0.05, evaluate=False),
-                   evaluate=False)
-    try:
-        Float(expr)
-        return v.doit()
-    except (TypeError, ValueError):
-        return v
-
-
-def eval_taylor_cos(expr):
```

Comment:
note to myself
Even in sigle precision , still callin cos instead of cosf (and sinf/...)